graylog2-web-interface/src/components/alertconditions/AlertConditionForm.jsx

```diff
@@ -10,20 +10,37 @@ const AlertConditionForm = React.createClass({
   },
   alertConditionsFactory: new AlertConditionsFactory(),
   getValue() {
-    return this.refs.conditionForm.getValue();
+    return {
+      title: this.refs.title.value,
+      parameters: this.refs.conditionForm.getValue(),
+    };
   },
+  getDefaultProps() {
+    return {
+      alertCondition: {
+        parameters: {},
+      }
+    };
+  },
+
   _formatConditionFormFields(type) {
     const typeDefinition = this.alertConditionsFactory.get(type);
 
     if (typeDefinition !== undefined) {
-      return <typeDefinition.configuration_form ref="conditionForm" alertCondition={this.props.alertCondition}/>;
+      return <typeDefinition.configuration_form ref="conditionForm" alertCondition={this.props.alertCondition.parameters}/>;
     }
 
     return undefined;
   },
   render() {
+    const alertCondition = this.props.alertCondition || {parameters: {}};
     return (
       <Well className="alert-type-form alert-type-form-message-count form-inline well-sm">
+        Title: <input ref="title" type="text" className="form-control" autoComplete="off" defaultValue={alertCondition.title}/>
+        {' '}
+        <small>This title can also be included in alert notifications.</small>
```

Comment:
I find this text a little confusing inside this form, as we mostly describe there how the condition should apply. I think adding some colour would help with that, I'll commit that so you can take a look.